packages/lib-user/src/components/shared/ProfileHeader/ProfileHeader.js

```diff
@@ -45,12 +49,12 @@ TitledStat.propTypes = {
 
 function ProfileHeader ({
   avatar = '',
-  classifications = 0,
-  contributors = 0,
+  classifications = undefined,
```

Comment:
Similar to my above question - why are these props initialized as `undefined` instead of `0`? The prop type is listed as a `number` for each.

While running the app locally, I noticed that each TitledStat component doesn't render until my `user` data is fetched. Is that behavior intended to handle ProfileHeader when no user exists (signed-out, or slow fetch from panoptes)? Just curious if the UX could instead be such that the labels "Classifications", "Hours", "Contributors", and "Projects" are statically rendered, but the `value` for each renders only once `user` data is fetched.

Reply:
The thought process behind the `undefined` default is that sometimes we want to show Classifications, but not Hours, vice versa, or Contributors or not, and once we know what we want to show, we want to show it with a value or even if it's falsey `0`. The undefined prevents TitleStats we don't want showing from showing, and shows `0` if applicable.

I'll think about other options/approaches and check with Sean on loading behavior.